extendr-api/src/robj/try_from_robj.rs

```diff
@@ -1,4 +1,4 @@
-//! Conversions to Robj
+//! Conversions to [`Robj`]
```

Comment:
Forgive my ignorance, but what do the square brackets do here?

Reply:
I believe they make a link in the docs that leads to `Robj`. You can see if the link works
in vscode, as it will also become a Ctrl/Cmd+Click link there as well (with Rust Analyzer).